commit 51e820fc08ad751f74a5329c82753aed985a4eec
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-07-22T15:11:21+01:00 (3 months ago)
    
    TOOLS-2287 bits-upload gets confused when building a gerrit branch
    Reviewed by: John Levon <john.levon@joyent.com>

diff --git a/tools/mk/Makefile.targ b/tools/mk/Makefile.targ
index a6f42d7..755107f 100644
--- a/tools/mk/Makefile.targ
+++ b/tools/mk/Makefile.targ
@@ -535,7 +535,7 @@ ifndef ENGBLD_BITS_UPLOAD_OVERRIDE
 .PHONY: bits-upload
 bits-upload: publish $(BUILDIMAGE_TARG)
 	$(TOP)/deps/eng/tools/bits-upload.sh \
-		-b $(BRANCH) \
+		-b "$(BRANCH)" \
 		-B "$(TRY_BRANCH)" \
 		$(BITS_UPLOAD_LOCAL_ARG) \
 		$(BITS_UPLOAD_IMGAPI_ARG) \
@@ -552,7 +552,7 @@ bits-upload: publish $(BUILDIMAGE_TARG)
 .PHONY: bits-upload-latest
 bits-upload-latest:
 	$(TOP)/deps/eng/tools/bits-upload.sh \
-		-b $(BRANCH) \
+		-b "$(BRANCH)" \
 		-B "$(TRY_BRANCH)" \
 		$(BITS_UPLOAD_LOCAL_ARG) \
 		$(BITS_UPLOAD_IMGAPI_ARG) \
